src/components/services-view/services-view.controller.ts

```diff
@@ -192,7 +192,7 @@ export class ServicesViewController implements angular.IController {
         this.ctrl.filterConfig.resultsCount = this.ctrl.filteredItems.length;
         $('.toolbar-pf-results h5').text(this.ctrl.filterConfig.resultsCount + ' of ' + this.ctrl.totalCount + ' items');
       } else {
-        $('.toolbar-pf-results h5').text(this.ctrl.totalCount + ' items');
+        $('.toolbar-pf-results h5').text(this.ctrl.totalCount + ' item' + (this.ctrl.totalCount > 1 ? 's' : ''));
```

Comment:
Do we ever need to worry about "0 items", which should be plural? I guess the subcategory would be hidden, but in case that ever changes, suggest instead `this.ctrl.totalCount !== 1`